Cargo.toml

```diff
@@ -7,6 +7,7 @@ edition = "2021"
 
 [package]
 name = "rustc_apfloat"
+license = "Apache-2.0 WITH LLVM-exception"
```

Comment:
Funny thing about doing it like this (and not on the workspace + `license.workspace = true`) is that it doesn't apply to the `rustc_apfloat-fuzz` crate - is that intentional?

Which reminds me, `fuzz/Cargo.toml` should have `publish = false`, which would make its metadata fields more obviously unimportant.

Though looking at [a different repo with `publish = false` crates](https://github.com/EmbarkStudios/rust-gpu/blob/e830e608eb9556ea914c5482addb4b7ca5f04e74/tests/deps-helper/Cargo.toml#L5-L9), it does seem good practice to fill in the fields even w/ `publish = false` (and it's pretty neat when using the new `.workspace = true` feature).